web/text_layer_builder.js

```diff
@@ -181,44 +184,116 @@ class TextLayerBuilder {
    * clicked. This reduces flickering of the content if the mouse is slowly
    * dragged up or down.
    */
-  #bindMouse() {
+  #bindMouse(end) {
     const { div } = this;
 
+    let isFirefox;
+    let selectingThroughMouse = false;
+    let prevRange = null;
```

Comment:
Can we please avoid "dead code" in Firefox, by doing something like the following?
```suggestion
    let selectingThroughMouse = false;
    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
      // eslint-disable-next-line no-var
      var isFirefox, prevRange = null;
    }
```